service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinClientOptions.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.sdk.iot.service.digitaltwin;
+
+import com.microsoft.azure.sdk.iot.service.ProxyOptions;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Configurable options for all digital twin client operations. This options bundle is used by both
+ * {@link DigitalTwinClient} and {@link DigitalTwinAsyncClient}.
+ */
+@Builder
+public class DigitalTwinClientOptions
+{
+    protected static final Integer DEFAULT_HTTP_READ_TIMEOUT_MS = 24000; // 24 seconds
```

Comment:
How are we deciding this number?

Reply:
These are the defaults we had for earlier options objects. These values were based on some similar hardcoded timeout values that have been in place for a very long time, but were definitely arbitrary.

The idea behind these defaults was so that customers who don't use our options objects will still see roughly the same behavior from the SDK. They will still get read and connect timeouts like the old client used to give